src/components/Map/layers/ClusterLayer.js

```diff
@@ -107,11 +107,9 @@ export default class ClusterLayer {
           <div class="bubble">
             <div class="total">${utils.numberNotation(marker.total_people)}</div>
             ${this.generateBubble(marker.per_sector, marker.total_people, bubbleSize)}
-            ${marker.clustered || zoom >= 6 ? `
```

Comment:
@edbrett did you discuss this change with Elena? In Africa the names get a bit hard to distinguish at some zoom levels. I was thinking that they could show on hover for zoom levels < 6, and then show permanently for zoom levels >= 6. (the actual number might require tweaking).